Bottom: 0f3f2366f2163f7b7685242f674386a7a7483059
Top:    40ecc6a5e52cc91795cc2027dbf7bb8f79643014
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-24 16:30:47 -0500

Remove libicu52 from Dockerfile

Newest upstream versions of php:5.6-apache are based on debian stretch and 
use libicu57, which is already installed. Besides this, libicu-dev would 
already install libicuXX.


---
diff --git a/docker/docker-mantisbt/Dockerfile b/docker/docker-mantisbt/Dockerfile
index b59a97c..496a84f 100644
--- a/docker/docker-mantisbt/Dockerfile
+++ b/docker/docker-mantisbt/Dockerfile
@@ -12,7 +12,6 @@ RUN sed -i 's| main$| main contrib non-free|' /etc/apt/sources.list \
     && apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
         libcurl3 \
-        libicu52 \
         libjpeg62-turbo \
         libmcrypt4 \
         libpq5 \
diff --git a/docker/docker-mediawiki/Dockerfile b/docker/docker-mediawiki/Dockerfile
index 6aaefd6..5442116 100644
--- a/docker/docker-mediawiki/Dockerfile
+++ b/docker/docker-mediawiki/Dockerfile
@@ -13,7 +13,6 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
         git \
         imagemagick \
-        libicu52 \
         libicu-dev \
         libldap2-dev \
     && \
